main.go

```diff
@@ -326,7 +327,6 @@ func (p *gateway) setupDebugServer(conf *config.GatewayConfig, logger log.Logger
 	})
 	p.debugServer.Mux.Handle("/debug/dims", &p.debugSink)
 
-	p.debugServer.Exp2.Exported["config"] = conf.Var()
```

Comment:
This needs to stay.

Reply:
I'm curious to understand why this needs to stay?

Personal, I don't see the benefit since it doesn't reflect the health of the app and also contains the raw access token